launcher/modplatform/atlauncher/ATLPackInstallTask.cpp

```diff
@@ -143,6 +169,116 @@ void PackInstallTask::onDownloadFailed(QString reason)
     emitFailed(reason);
 }
 
+void PackInstallTask::deleteExistingFiles()
+{
+    setStatus(tr("Deleting existing files..."));
+
+    // Setup defaults, as per https://wiki.atlauncher.com/pack-admin/xml/delete
+    VersionDeletes deletes;
+    deletes.folders.append(VersionDelete{ "root", "mods%s%" });
```

Comment:
I think Qt handles `/` just fine, don't think we need to insert the right path separator here

Reply:
It does.

They are replaced elsewhere in the code. I've matched the ATLauncher here, as per the above - but also it's the format the API uses, so it's somewhat easier to compare when looking at both.